ee/query-service/app/api/gateway.go

```diff
@@ -9,7 +9,15 @@ import (
 
 func (ah *APIHandler) ServeGatewayHTTP(rw http.ResponseWriter, req *http.Request) {
 	ctx := req.Context()
-	if !strings.HasPrefix(req.URL.Path, gateway.RoutePrefix+gateway.AllowedPrefix) {
```

Comment:
This proxy is for the gateway. Hence the full url prefix becomes `/api/gateway/v1/workspaces/me`.

Was your plan to add the profile endpoint in this only? i.e. `/api/gateway/v2/profiles/me`.

Reply:
> Was your plan to add the profile endpoint in this only? i.e. /api/gateway/v2/profiles/me.
Yes